coding-concepts.js

```diff
@@ -8,36 +8,36 @@
 // --------------------1) What will this log?
 
 const cohort = "Hotel 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: [hotel , 2022] because (.split()) method turns strings into arrays
+// b) Verify and explain: ['H', 'o', 't', 'e','l', ' ', '2', '0','2', '2' ] i was kinda right turns the string into the array but split every single letter
```

Comment:
yes the format of the "" will determine how the values are displayed in the array